sdk/src/main/java/com/kin/ecosystem/Kin.java

```diff
@@ -74,27 +85,105 @@ private static Kin getInstance() {
 		return instance;
 	}
 
-	public static void start(@NonNull Context appContext, @NonNull WhitelistData whitelistData,
-		@NonNull KinEnvironment environment)
-		throws ClientException, BlockchainException {
+	public synchronized static void initialize(Context appContext) throws ClientException {
```

Comment:
add javadocs explaining the use (and also reffering to login)

Reply:
👍

Reply:
can we delete synchronized keyword ?

Reply:
no